content/en/docs/languages/cpp/basics.md

```diff
@@ -38,6 +38,13 @@ Get the example code and build gRPC:
     ```sh
     $ cd examples/cpp/route_guide
     ```
+ 3. Prepare the cmake environment so that you can build it easily.
+
+    ```sh
+    $ mkdir -p cmake/build
+    $ pushd cmake/build
+    $ cmake -DCMAKE_PREFIX_PATH=$MY_INSTALL_DIR ../..
```

Comment:
Add a `popd` here?

Reply:
Good catch! Oh it should use `cd` instead of `pushd` so that following commands can run on cmake/build directory.